### PR TITLE
CLF overhaul

### PR DESCRIPTION
**Pull request by stanalbatross**
## About The Pull Request

Overhauls the CLF loadouts (apart from synth and cell commander) with more up-to-date stuff plus nicer guns

## Why It's Good For The Game

Makes CLF not literal dogshit to fight as, gives them a bit of a chance against xenos.

## Changelog

:cl:
balance: overhauled the CLF loadouts, giving more/better weapons, removing the mod 88 from the rifle pool, and giving the spec a chance to spawn with an r4t! The spec also gets an anti-tank rocket launcher and maybe an LMG. Other loadouts all get some other tweaks.
/:cl:

### CODE REVIEW

File: code/modules/gear_presets/_select_equipment.dm [hunk 1]
```diff
@@ -395,8 +397,9 @@
 	if(!M) return
 
 	var/list/rebel_gunner_firearms = list(
-		/obj/item/weapon/gun/shotgun/merc = /obj/item/ammo_magazine/handful/shotgun/slug,
+	//	/obj/item/weapon/gun/shotgun/merc = /obj/item/ammo_magazine/handful/shotgun/slug, //no. This gun is poo poo.
```
Comment (Geevies) [inline code/modules/gear_presets/_select_equipment.dm:400]:
```suggestion
```

File: code/modules/gear_presets/_select_equipment.dm [hunk 2]
```diff
@@ -407,6 +410,70 @@
...
+
+	spawn_weapon(gunpath, ammopath, M, ammo_amount)
+
+	return 1
+
+/datum/equipment_preset/proc/spawn_rebel_rifle(var/atom/M, var/ammo_amount = 12)
+	if(!M) return
+
+
+
```
Comment (Geevies) [inline code/modules/gear_presets/_select_equipment.dm:469]:
```suggestion
```

File: code/modules/gear_presets/_select_equipment.dm [hunk 3]
```diff
@@ -407,6 +410,70 @@
...
+	if(!M) return
+
+	var/gunpath = pick(rebel_smgs)
+	var/ammopath = rebel_smgs[gunpath]
+
+	spawn_weapon(gunpath, ammopath, M, ammo_amount)
+
+	return 1
+
+
```
Comment (Geevies) [inline code/modules/gear_presets/_select_equipment.dm:454]:
```suggestion
```

File: code/modules/gear_presets/clf.dm [hunk 4]
```diff
@@ -165,29 +174,42 @@
 
 /datum/equipment_preset/clf/specialist/load_gear(mob/living/carbon/human/H)
 
-	//specific armor for specialist
-	H.equip_to_slot_or_del(new /obj/item/clothing/under/colonist/clf(H), WEAR_BODY)
+	//jumpsuit and their webbing
+	var/obj/item/clothing/under/colonist/clf/CLF = new()
+	var/obj/item/clothing/accessory/storage/webbing/W = new()
+	CLF.attach_accessory(H, W)
+	W.hold.storage_slots = 5
```
Comment (Geevies) [inline code/modules/gear_presets/clf.dm:181]: make a special variant rather than adjusting the slots

File: code/modules/gear_presets/clf.dm [hunk 5]
```diff
@@ -268,26 +293,31 @@
 	H.set_species(SYNTH_COLONY)
 
 /datum/equipment_preset/clf/synth/load_gear(mob/living/carbon/human/H)
+
```
Comment (Geevies) [inline code/modules/gear_presets/clf.dm:296]:
```suggestion
```

**Review by Geevies — CHANGES_REQUESTED** [review]

File: code/modules/gear_presets/clf.dm (hunk 4 above)
Comment (stanalbatross) [reply code/modules/gear_presets/clf.dm]: done, also cleaned up some UPP stuff that did the same thing

**Comment by Nanu308** [comment]
Incendiary Ammo in some of the shotguns might end up being too strong, but it can easily be swapped out later on.

File: code/modules/gear_presets/_select_equipment.dm [hunk 6]
```diff
@@ -307,6 +307,7 @@
 		/obj/item/storage/backpack/general_belt,
 		/obj/item/storage/large_holster/katana/full,
 		/obj/item/storage/large_holster/machete/full,
+		/obj/item/storage/belt/marine,
```
Comment (Shad0vvs) [inline code/modules/gear_presets/_select_equipment.dm:310]:
```suggestion
		/obj/item/storage/belt/marine
```

File: code/modules/gear_presets/_select_equipment.dm [hunk 7]
```diff
@@ -336,7 +339,6 @@
 		/obj/item/weapon/gun/pistol/skorpion = /obj/item/ammo_magazine/pistol/skorpion,
 		/obj/item/weapon/gun/pistol/skorpion/upp = /obj/item/ammo_magazine/pistol/skorpion,
 		/obj/item/weapon/gun/smg/uzi = /obj/item/ammo_magazine/smg/uzi,
```
Comment (Shad0vvs) [inline code/modules/gear_presets/_select_equipment.dm:341]:
```suggestion
		/obj/item/weapon/gun/smg/uzi = /obj/item/ammo_magazine/smg/uzi
```

File: code/modules/gear_presets/_select_equipment.dm [hunk 8]
```diff
@@ -407,6 +409,67 @@
...
 	return 1
 
+
+var/list/rebel_shotguns = list(
+	/obj/item/weapon/gun/shotgun/double = /obj/item/ammo_magazine/handful/shotgun/buckshot,
+	/obj/item/weapon/gun/shotgun/double = /obj/item/ammo_magazine/handful/shotgun/flechette,
+	/obj/item/weapon/gun/shotgun/pump/cmb = /obj/item/ammo_magazine/handful/shotgun/incendiary,
+	/obj/item/weapon/gun/shotgun/pump/cmb = /obj/item/ammo_magazine/handful/shotgun/incendiary,
+	/obj/item/weapon/gun/shotgun/double/sawn = /obj/item/ammo_magazine/handful/shotgun/incendiary,
+	/obj/item/weapon/gun/shotgun/double/sawn = /obj/item/ammo_magazine/handful/shotgun/buckshot,
```
Comment (Shad0vvs) [inline code/modules/gear_presets/_select_equipment.dm:419]:
```suggestion
	/obj/item/weapon/gun/shotgun/double/sawn = /obj/item/ammo_magazine/handful/shotgun/buckshot
```

**Review by Shad0vvs — APPROVED** [review]

**Pull request merged**